prisma/schema.prisma

```diff
@@ -216,8 +216,8 @@ model Availability {
   eventType   EventType? @relation(fields: [eventTypeId], references: [id])
   eventTypeId Int?
   days        Int[]
-  startTime   Int
-  endTime     Int
+  startTime   DateTime   @db.Time
+  endTime     DateTime   @db.Time
```

Comment:
Did you decide not to go with `@db.Timetz(x)`?

Please add some comments inline here in the schema with instructions on how we should consider timezones when we work with it

Reply:
I did further research after our conversation and learned that Timetz is a "Don't use" field in psql. The reason is that it is "fundamentally flawed" in the eyes of the PostgreSQL team. 

There is some truth in that at a timetz without knowing the date actually does not tell you what time it is; only by knowing the current day can you know which time it is.